src/errors.js

```diff
@@ -10,16 +10,9 @@ class SchemaError extends Error {
     }
 
     this.name = 'SchemaError'
-    this._node = node
-  }
-
-  toString() {
```

Comment:
This is not always used, for example mocha does not use it when outputting errors. So by setting the message property the base class's `toString()` still works if called and so does just using `error.message` to print an error.